packages/vite/src/node/plugins/html.ts

```diff
@@ -1182,6 +1182,7 @@ export function injectNonceAttributeTagHook(
 
       if (
         node.nodeName === 'script' ||
+        node.nodeName === 'style' ||
```

Comment:
The original commit where this was removed was doing:
```ts
(node.nodeName === 'style' && node.childNodes.length)
```

Am I correct in assuming that this is excluding the case where there is no content such as: `<style></style>`? If that's the case its unclear to me why we would be excluding this as I think the browser would still probably fail on this and require `'unsafe-inline'`.

Reply:
Yeah, I think the condition should be without `node.childNodes.length`.